peertube/README.md

```diff
@@ -0,0 +1,70 @@
+# peertube
+
+[Peertube](https://github.com/Chocobozzz/PeerTube) is a free, decentralized
+and federated video platform developed as an alternative to other platforms
+that centralize our data and attention, such as YouTube, Dailymotion or Vimeo.
+
+Peertube is a video streaming service, and while you don't need super beefy
+hardware to run it, you do want hardware that can handle it. Read Peertube's
+hardware recommendations [here](https://joinpeertube.org/faq#should-i-have-a-big-server-to-run-peertube).
+
+Also, keep in mind that videos take up a lot of space. You may want to consider
+increasing your host's storage or symlinking `/var/lib/docker/volumes/peertube_data/`
+on the host to a mountpoint for external storage.
+
+## Config
+
+```
+make config
+```
+
+This will ask you to enter the domain name to use.
+It automatically saves your responses into the configuration file
+`.env_{INSTANCE}`.
+
+### Authentication and Authorization
```

Comment:
I know its probably important info, but its the same info as all the other apps right? It probably shouldn't be duplicated in every app README, but maybe we could have a separate document that explains this one time.

Reply:
I used whoami as a template. until we improve the template project, I'll have to remember to clean the readme better.

fixed in [a3a8e3a](https://github.com/EnigmaCurry/d.rymcg.tech/pull/260/commits/a3a8e3aff9db0591c542a3b2cd8f8b0a6c0a42f1)